### DISCUSSION TITLE
Welcome to PKMDS-Blazor Discussions!

### DISCUSSION THREAD

**Discussion by codemonkey85** [codemonkey85/PKMDS-Blazor#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Krimdra** [comment]
Hello I'm Krimdra (26 M) and I was wondering if this discussion board is open... I wanted to talk about the PKMDS site and whether or not there might updates to it.

**Reply by codemonkey85** [reply]
Sure, any feedback you want to leave here is fine.
